CONTRIBUTING.md

```diff
@@ -17,9 +17,9 @@
 * `ember test --server` – Runs the test suite in "watch mode"
```

Comment:
Let's prefix these commands with `cd test-app && ` so they can be invoked from root folder sucessfully.

Also noted `cd my-addon` on line 6 should be updated to `cd ember-css-transitions`